include/envoy/http/access_log.h

```diff
@@ -114,6 +114,17 @@ class RequestInfo {
    * Set whether the request is a health check request or not.
    */
   virtual void healthCheck(bool is_hc) PURE;
+
+  /**
+   * Set the downstream address
+   */
+  virtual void downStreamAddress(std::string address) PURE;
```

Comment:
Setting aside the formatting of getter/setter, I don't think there should be a setter on this interface for this. It doesn't really make sense as the address is fixed. Unclear why it would need to be changed.